docs/rules/prefer-strict-equal.md

```diff
@@ -0,0 +1,25 @@
+# Suggest using `toStrictEqual()` (prefer-strict-equal)
+
+`toStrictEqual` not only checks that two objects contain the same data but also
+that they have the same shape. The believe is that imposing a stricter equality
```

Comment:
"The believe is that" sounds odd. Could you re-phrase?

Reply:
I tweaked and expanded on this a bit in a7ca3b2